system_metrics_collector/src/system_metrics_collector/collector.cpp

```diff
@@ -52,6 +52,7 @@ bool Collector::Stop()
 
 void Collector::AcceptData(const double measurement)
 {
+  // todo <dabonnie> should this throw if not started?
```

Comment:
The convention we use is `// TODO(username)`, however should we keep this todo here? It doesn't seem in scope for this review.